app/views/shared/search/_other.html.erb

```diff
@@ -1,29 +1,4 @@
-<% unless @no_shared_search %>
-  <% if (controller_name == 'commodities' && action_name == 'show') || (controller_name == 'headings' && action_name == 'show' && @heading&.declarable?) %>
-    <header>
-      <h1 class="govuk-heading-l commodity-header" data-comm-code="<%= commodity_code %>">
-        Commodity information for <%= commodity_code %>
-        <div class="copy_code" id="copy_code">
-          <span id="copy_comm_code" class="pseudo-link govuk-link">Copy commodity code</span><br>
-          <span class="copied">Code copied</span>
-        </div>
-      </h1>
-
-      <%= form_tag perform_search_path, method: :get, class: "tariff-search #{@section_css}", id: "new_search" do |f| %>
-        <div class="govuk-grid-row">
-          <div class="govuk-grid-column-full">
-            <div class="search-header">
-              <div class="searchfield govuk-form-group">
-                <%= render 'shared/search_date_picker' %>
-                <%= render 'shared/currency_picker' %>
-              </div>
-            </div>
-          </div>
-        </div>
-      <% end %>
-    </header>
-  <% elsif %w(search search_references exchange_rates pages errors feedback).exclude?(controller_name) && action_name != 'quota_search' %>
-    <%= page_header heading_for(section: @section,chapter: @chapter,heading: @heading, commodity: @commodity) %>
```

Comment:
I think the only condition we are considering is the 
`action_name != 'quota_search'`